docs/source/usage/generating_metadata.rst

```diff
@@ -0,0 +1,149 @@
+Generating Metadata
+====================
+
+One of the main functionalities of MetaSynth is the functionality to create a metadata representation of a given dataset, resulting in a :obj:`MetaDataset <metasynth.dataset.MetaDataset>` object. 
+This object captures essential aspects of the dataset, including variable names, types, data types, the percentage of missing values, and distribution attributes.
+
+.. image:: /images/flow_metadata_generation.png
+   :alt: Metadata_generation_flowchart
+
+:obj:`MetaDatasets <metasynth.dataset.MetaDataset>` follow the  `Generative Metadata Format
+(GMF) <https://github.com/sodascience/generative_metadata_format>`__, a standard designed to be easy to read and understand. 
+This metadata can be exported as a .JSON file, allowing for manual and automatic editing, as well as easy sharing.
+
+
+.. raw:: html
+
+   <details> 
+   <summary> An example of a metadataset: </summary>
+
+.. code-block:: json
```

Comment:
We should probably update this file with the current version. Maybe an issue can be made?

Reply:
Yep, agreed, I had post-poned it just in case it would change again, but should be done.